content/en/docs/tutorials/pod-security/cluster-level-pss.md

```diff
@@ -0,0 +1,271 @@
+---
+title: Applying Pod Security Standards at Cluster level
+content_type: tutorial
+weight: 10
+---
+
+{{% alert title="Disclaimer" %}}
+This tutorial applies only for new clusters.
+{{% /alert %}}
+
+# Pre-requisites
+
+Install the following on your workstation:
+
+- [KinD](https://kind.sigs.k8s.io/docs/user/quick-start/#installation)
```

Comment:
Are these the only two things needed to fully complete this tutorial? Nothing else?

Reply:
I think so. I have asked one of our newer members to test run this tutorial to confirm!!

Reply:
Closing for now. This list also checks out with blog's pre-requisite: https://github.com/kubernetes/website/pull/30502/files#r752655898